pages/linux/qtchooser.md

```diff
@@ -0,0 +1,24 @@
+# qtchooser
+
+> A wrapper used to select between Qt development binary versions.
+> More information: <https://manned.org/qtchooser>.
+
+- List available Qt versions from the configuration files:
+
+`qtchooser --list-versions`
+
+- Print environment information:
+
+`qtchooser --print-env`
+
+- Run the specified tool using the specified Qt version:
```

Comment:
```suggestion
- Run the specified tool using a specific Qt version:
```

Reply:
I think that's just a matter of preference. Both seem fine to me.

Reply:
The two version are completely equivalent and correct, so much so that I prefer neither.

Reply:
Once this is resolved ill give my approval

Reply:
I can't say I hold an opinion on this one.
I made my partner judge without telling her who did which, she vouched for "the specified". ^-^' Idk if that's worth much.

Overall, I think it makes little difference.

Reply:
I'll resolve.